projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field.component.html

```diff
@@ -45,7 +45,8 @@ <h2 class="govuk-heading-l">Case file</h2>
                          [contentType]="currentDocument.content_type"
                          [enableAnnotations]="true"
                          [enableRedactions]="true"
-                         [height]="'94.5vh'">
+                         [height]="'94.5vh'"
+                         [enableICP]="isIcpEnabled(icpEnabled,icp_jurisdictions)">
```

Comment:
We shouldn't do this. We looked at the media viewer code and it doesn't work that way. I The isIcpEnabled function checks the current jurisdiction against the list so there's no need